process/data_processor.py

```diff
@@ -100,30 +101,38 @@ def fit_transform_subblocks(self, X_seq, term_weighting=None):
             # put into dataframe to add nas to missing events
             # divide by 20 as original operation multiplied by 20
             block_df = pd.DataFrame(block_counts, columns=self.events) / 20
+            block_df = block_df.reindex(sorted(block_df.columns), axis=1)
             block_df = block_df.fillna(0)
+            if self.rolling:
```

Comment:
I started by putting it here since we were already touching the data frames

but if care about optimizing it we could move some stuff around